src/features/account/components/AccountRoleTags.tsx

```diff
@@ -0,0 +1,81 @@
+import { Tag, TagLabel, TagLeftIcon, HStack } from '@chakra-ui/react';
+import { Trans } from '@lingui/macro';
+import type { FunctionComponent } from 'react';
+
+import { AuditorIcon, DeveloperIcon, ReviewerIcon } from '../../../components';
+
+export enum AccountRole {
+  DEVELOPER = 'developer',
+  AUDITOR = 'auditor',
+  REVIEWER = 'reviewer',
+}
+
+export type AccountRoleTagProps = {
+  roles: AccountRole[];
+};
+
+export const AccountRoleTags: FunctionComponent<AccountRoleTagProps> = ({
+  roles,
+}) => {
+  const roleset = new Set(roles);
+  const roleArr = Array.from(roleset);
+
+  return (
+    <HStack spacing={2} alignItems="center">
+      {roleArr.map((role, i) => {
+        switch (role) {
+          case AccountRole.DEVELOPER:
```

Comment:
These components seem to be largely the same. It seems easier to define the values in a separate object, and use the role as object keys.

```ts
const ACCOUNT_ROLES = {
  [AccountRole.Developer]: {
    background: 'some.theme.value',
    color: 'some.other.theme.value',
  },
  // etc.
};
```